tests/TestCase/Database/Schema/SchemaDialectTest.php

```diff
@@ -167,4 +168,59 @@ public function testDescribeOptionsMysql(): void
         $this->assertArrayHasKey('engine', $result);
         $this->assertArrayHasKey('collation', $result);
     }
+
+    public function testHasColumn(): void
+    {
+        $this->assertFalse($this->dialect->hasColumn('orders', 'nope'));
+        $this->assertFalse($this->dialect->hasColumn('orders', ''));
+        $this->assertFalse($this->dialect->hasColumn('invalid', 'also invalid'));
+
+        $this->assertTrue($this->dialect->hasColumn('users', 'username'));
+        $this->assertFalse($this->dialect->hasColumn('users', 'USERNAME'));
```

Comment:
Technically, all unquoted columns are lower case in postgresql, but I think this is correct here.

Reply:
True that postgres would be ok with this column name sometimes. I wanted schema reflection to use the column names as defined in the schema though.

Reply:
I think users will have to use the correct quoting or lower case to match and that's ok. My_Column is converted to my_column and they should deal with it.